modules/gapi/test/common/gapi_video_tests_common.hpp

```diff
@@ -321,6 +322,53 @@ inline GComputation runOCVnGAPIOptFlowPipeline(TestFunctional& testInst,
     return c;
 }
 
+inline void compareBackSubResults(const cv::Mat &actual, const cv::Mat &expected, const int diffPercent)
+{
+    GAPI_Assert(actual.size() == expected.size());
+    int allowedNumDiffPixels = actual.size().area() * diffPercent / 100;
+
+    cv::Mat diff;
+    cv::absdiff(actual, expected, diff);
+
+    cv::Mat hist(256, 1, CV_32FC1, cv::Scalar(0));
+    const float range[]{ 0, 256 };
+    const float *histRange{ range };
+    calcHist(&diff, 1, 0, Mat(), hist, 1, &hist.rows, &histRange, true, false);
+    for (int i = 2; i < hist.rows; ++i)
+    {
+        hist.at<float>(i) += hist.at<float>(i - 1);
+    }
+
+    int numDiffPixels = static_cast<int>(hist.at<float>(255));
+
+    EXPECT_GT(allowedNumDiffPixels, numDiffPixels);
+}
+
+inline void testBackSubInStreaming(cv::GStreamingCompiled gapiBackSub, const int diffPercent)
+{
+    cv::Mat frame,
+        gapiForeground,
+        ocvForeground;
+
+    gapiBackSub.start();
+    EXPECT_TRUE(gapiBackSub.running());
+
+    // OpenCV reference substractor
+    auto pOCVBackSub = createBackgroundSubtractorMOG2();
```

Comment:
Would you mind to use a reference for namespace here? `cv::createBackgroundSubtractorMOG2`

Reply:
For what? It works in this view. Why do we need extra characters?

Reply:
Please, this is for consistency.